mobify/progressive-web-scaffold
# Make cart full height in app

When in an app, the header and footer don't show, so we want to show the empty cart UI full-screen.

 **JIRA**: https://mobify.atlassian.net/browse/HYB-1018

## Changes
- Add class for full viewport height
- Use that class on empty cart if running in Astro

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Run in app
- Go to cart, should be full screen
- Run in web
- Go to cart, should not be full screen
